framer/LayerStyle.coffee

```diff
@@ -90,6 +90,11 @@ exports.LayerStyle =
 
 		return css.join(" ")
 
+	_roundToZero: (num) ->
```

Comment:
Can you please make this a standalone function. We sometimes want to walk through the keys of LayerStyles and we expect them all to be css properties.